common/Perf/Azure.Test.Perf/PerfProgram.cs

```diff
@@ -31,6 +32,13 @@ public static class PerfProgram
 
         public static async Task Main(Assembly assembly, string[] args)
         {
+            // See if we want to run a BenchmarkDotNet microbenchmark
+            if (args.Length > 0 && args[0].Equals("micro", StringComparison.OrdinalIgnoreCase))
```

Comment:
This is the change to allow running BDN tests through the Perf runner using the `dotnet run -- micro ..` command line. 

TODO: discuss with the team if this is the way we want it.

cc @mikeharder

Reply:
This change looks fine to me.